FrontEnd/src/components/modules/login/LoginForm.tsx

```diff
@@ -0,0 +1,74 @@
+// LoginForm.tsx
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm, type SubmitHandler } from "react-hook-form";
+import { loginSchema, type LoginSchema } from "src/lib/schemas/loginSchema";
+import InputsLogin from "src/components/forms/InputsLogin";
+
+const LoginForm = () => {
+  const { control, handleSubmit, formState: { errors } } = useForm<LoginSchema>({
+    resolver: zodResolver(loginSchema), defaultValues: {
```

Comment:
```suggestion
resolver: zodResolver(loginSchema as any), defaultValues: {
```
resuelve problema del tipo de loginSchema ( de momento )